### PR TITLE
feat: added the nav bar

### PR DESCRIPTION
**Pull request by Adispandey**
Added the Nav bar which indicates the active state of the nav bar.

image

### CODE REVIEW

File: user-story/src/App.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+export const App = () => {
+  return (
+    <BrowserRouter>
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route path="/dashboard" element={<Dashboard />} />
+        <Route path="/info" element={<Info />} />
+        <Route path="/recent" element={<Recent />} />
+        <Route path="/setting" element={<Setting />} />
```
Comment (divija-zemoso) [inline user-story/src/App.tsx]: create a separate variables for routes and get the path value

File: user-story/src/components/organisms/NavBar/index.tsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  border: '1px solid black',
+  backgroundColor: '#9AC1E9',
+})
+const Content = styled(Stack)({
+  justifyContent: 'center',
+  marginLeft: '30%',
+})
+const NavBar = () => {
+  const navigate = useNavigate()
+  const [active, setActive] = useState<string>('Home')
```
Comment (divija-zemoso) [inline user-story/src/components/organisms/NavBar/index.tsx:23]: Access **Home** from SIDEBAR_DATA

File: user-story/src/components/organisms/NavBar/index.tsx [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+        navigate('/dashboard')
+        break
+      case INFO:
+        navigate('/info')
+        break
+      case RECENT:
+        navigate('/recent')
+        break
+      case SETTING:
+        navigate('/setting')
```
Comment (divija-zemoso) [inline user-story/src/components/organisms/NavBar/index.tsx]: create a route variables and access these paths from it

File: user-story/src/components/organisms/NavBar/index.tsx [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+            icon={<bar.icon />}
+            backgroundColor={active === bar.label ? '#4B0082' : '#9AC1E9'}
+            color={active === bar.label ? 'white' : ''}
+            handleClick={() => handleClick(bar.label)}
+          />
+        ))}
+      </NavContainer>
+      <Content>
+        <Typography variant="h2">{active}</Typography>
+      </Content>
```
Comment (divija-zemoso) [inline user-story/src/components/organisms/NavBar/index.tsx]: If it is being navigated to the respective pages what is the use of this code

File: user-story/src/components/organisms/NavBar/index.tsx [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+      case INFO:
+        navigate('/info')
+        break
+      case RECENT:
+        navigate('/recent')
+        break
+      case SETTING:
+        navigate('/setting')
+        break
+      default:
```
Comment (divija-zemoso) [inline user-story/src/components/organisms/NavBar/index.tsx]: separate functionality from the component

**Review by divija-zemoso — COMMENTED** [review]
-> please create a template for pages
reduce the code duplications

File: user-story/src/App.tsx (hunk 1 above)
Comment (Adispandey) [reply user-story/src/App.tsx]: Address

File: user-story/src/components/organisms/NavBar/index.tsx (hunk 2 above)
Comment (Adispandey) [reply user-story/src/components/organisms/NavBar/index.tsx:23]: Address

File: user-story/src/components/organisms/NavBar/index.tsx (hunk 3 above)
Comment (Adispandey) [reply user-story/src/components/organisms/NavBar/index.tsx]: Address

File: user-story/src/components/organisms/NavBar/index.tsx (hunk 4 above)
Comment (Adispandey) [reply user-story/src/components/organisms/NavBar/index.tsx]: Address

File: user-story/src/components/organisms/NavBar/index.tsx (hunk 5 above)
Comment (Adispandey) [reply user-story/src/components/organisms/NavBar/index.tsx]: Address

**Pull request merged**